File: Swapnali Mahadik/C codes/GCD.c
#include<stdio.h>
int main()
{
    int num1,num2,gcd,temp;

    printf("Enter a Num1 :");
    scanf("%d",&num1);
    printf("Enter a Num2 :");
    scanf("%d",&num2);

    if(num1<num2)  //num1=20 ,num2=24
    {
        temp=num1;
        num1=num2;
        num2=temp;
    }//num1=24,num2=20

    while(num2 !=0)
    {
        temp=num1 %num2;
        num1=num2;  //num=20,4
        num2=temp;  //4,0
    }
    gcd=num1;//4

    printf("GCD is: %d",gcd);

    return 0;
}

File: Siddhant Gedekar/Java_Codes/java Codes/GCD.java
public class GCD {
    public static void main(String[] args) {
        int a = 24;
        int b = 59;
        int max=0;
        for(int i=2;i<24;i++){
            if(a%i==0 && b%i==0){
                max = i;
            }
        }
        System.out.println("GCD of "+a+" and "+b+" is "+max);
    }
}
